packages/core/types/src/payment/provider.ts

```diff
@@ -26,33 +37,26 @@ export type PaymentActions =
 /**
  * @interface
  *
- * Context data provided to the payment provider when authorizing a payment session.
+ * Context data provided to the payment provider
  */
 export type PaymentProviderContext = {
   /**
-   * The payment's billing address.
-   */
-  billing_address?: PaymentAddressDTO
-
-  /**
-   * The associated customer's email.
-   */
-  email?: string
-
-  /**
-   * The ID of payment session the provider payment is associated with.
+   * The account holder information, if available for the payment provider.
    */
-  session_id?: string
+  accountHolder?: PaymentAccountHolderDTO
```

Comment:
nit: Should probably be snake_cased to align with the general convention

Reply:
Yes good point 👍